x-pack/plugins/apm/server/lib/services/get_service_transaction_groups.ts

```diff
@@ -126,36 +123,21 @@ export async function getTransactionGroupsForPage({
           value: bucket.doc_count,
         }),
         errorRate,
+        impact: totalDuration
+          ? (transactionGroupTotalDuration * 100) / totalDuration
+          : 0,
       };
     }) ?? [];
 
-  const totalDurationValues = transactionGroups.map(
-    (group) => (group.latency ?? 0) * group.throughput
-  );
-
-  const minTotalDuration = Math.min(...totalDurationValues);
-  const maxTotalDuration = Math.max(...totalDurationValues);
-
-  const transactionGroupsWithImpact = transactionGroups.map((group) => ({
-    ...group,
-    impact:
-      (((group.latency ?? 0) * group.throughput - minTotalDuration) /
-        (maxTotalDuration - minTotalDuration)) *
-      100,
-  }));
-
-  // Sort transaction groups first, and only get timeseries for data in view.
-  // This is to limit the possibility of creating too many buckets.
-
-  const sortedAndSlicedTransactionGroups = orderBy(
-    transactionGroupsWithImpact,
-    sortField,
-    [sortDirection]
-  ).slice(pageIndex * size, pageIndex * size + size);
+  // By default sorts transactions by impact
+  const sortedTransactionGroups = sortBy(transactionGroups, 'impact').reverse();
 
   return {
-    transactionGroups: sortedAndSlicedTransactionGroups,
-    totalTransactionGroups: transactionGroups.length,
+    requestId: uuid(),
```

Comment:
I would have expected this to be set in the browser, as it is really specific to how the browser should fetch data, and not related to this API call.